### DISCUSSION TITLE
Add doses display for butterfly / moth mixes to the Item Charges plugin

### DISCUSSION THREAD

**Discussion by casper20051** [runelite/runelite#17999] [Ideas]
Currently the Item Charges plugin only displays doses for potions and not for butterfly mixes and moth mixes.

**Comment by YvesW** [comment]
Related PR for identification: https://github.com/runelite/runelite/pull/17626
Related PR for item stats: https://github.com/runelite/runelite/pull/17607
For item charges, no PR exists yet.
